tests/modules/nf-core/whamg/test.yml

```diff
@@ -1,6 +1,10 @@
 - name: whamg test_whamg_bam
-  command: nextflow run ./tests/modules/nf-core/whamg -entry test_whamg_bam -c ./tests/config/nextflow.config  -c ./tests/modules/nf-core/whamg/nextflow.config
+  command: nextflow run ./tests/modules/nf-core/whamg -entry test_whamg_bam -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/whamg/nextflow.config
   tags:
     - whamg
   files:
     - path: output/whamg/test.vcf.gz
+      md5sum: 35c622fb7f1c8306fdadd11cc9fa833a
+    - path: output/whamg/test.vcf.gz.tbi
+      md5sum: e7180bb953d2bd657c420a5f76a7164d
+    - path: output/whamg/versions.yml
```

Comment:
Do we need this?

Reply:
What do you mean exactly?

Reply:
Sorry, probably naive question- but is it conventional to have the versions file in the test.yml? Or, actually, is this different to the regular versions file?

Reply:
Ah yes! The tests need to check for the presence of the `versions.yml` file only (this is also automatically generated using `nf-core modules create-test-yml`)

Reply:
I guess what I'm getting at is that output/whamg/versions.yml isn't in the output files (though versions.yml is), so is it important to check its presence in the output?

Reply:
I don't really follow your reasoning here sorry, `versions.yml` is in the `output/whamg` folder

Reply:
Sorry, just me having a dim moment, never mind

Reply:
No problem, happens to the best of us :p